modules/audit_config/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "service" {
+  description = "Service which will be enabled for audit logging" 
+  type        = string
+  default     = "allServices"
+}
+
+variable "audit_log_config" {
+  description = "Map of log type and exempted members to be addded to service"
+  type        = list(map(string))
+}
+
+variable "project" {
+  description = "Project to add the IAM policies/bindings"
+  default     = ""
```

Comment:
We should never have invalid defaults. If a project is required, don't set a default.